rr_platform/msg/race_reset.msg

```diff
@@ -0,0 +1 @@
+# Message for reseting the state in the navigation controller
```

Comment:
@barulicm @chsahit @ebretl do we want to start requiring time stamps on messages as a convention?

Reply:
I don't think so. I saw that pop up on your repo, but honestly there are a lot of message types that we don't care about timestamping.
We're not going to be syncing this data up with any other sources, so the timestamp would never be used.